src/System.Private.CoreLib/shared/System/Threading/CancellationToken.cs

```diff
@@ -273,9 +273,7 @@ private CancellationTokenRegistration Register(Action<object?> callback, object?
                 throw new ArgumentNullException(nameof(callback));
 
             CancellationTokenSource? source = _source;
-            return source != null ?
-                source.InternalRegister(callback, state, useSynchronizationContext ? SynchronizationContext.Current : null, useExecutionContext ? ExecutionContext.Capture() : null) :
-                default; // Nothing to do for tokens than can never reach the canceled state. Give back a dummy registration.
+            return source?.InternalRegister(callback, state, useSynchronizationContext ? SynchronizationContext.Current : null, useExecutionContext ? ExecutionContext.Capture() : null) ?? default; // Nothing to do for tokens than can never reach the canceled state. Give back a dummy registration.
```

Comment:
Please put the `default` and everything after it back on its own line.  This new formatting creates an incredibly long line.